styles.css

```diff
@@ -176,67 +132,82 @@ footer p:last-of-type{
 .soclist a:hover{
   opacity: 0.6;
 }
-.soc_fb, .soc_tw, .soc_gg, .soc_in, .soc_fl, .soc_pi, .soc_rs, .soc_em {
-  background: url('imgs/mastersprite.png');
-  width: 24px;
-  height:24px;
-  margin:0;
-}
-.soc_fb{
-  background-position: 0 0;
-}
-.soc_tw{
-  background-position: -24px 0;
-}
-.soc_gg{
-  background-position: -48px 0;
-}
-.soc_in{
-  background-position: -72px 0;
-}
-.soc_fl{
-  background-position: -96px 0;
-}
-.soc_pi{
-  background-position: -120px 0;
-}
-.soc_rs{
-  background-position: -144px 0;
-  margin-left: 10px;
-}
-.soc_em{
-  background-position: -168px 0;
+.soc_fb, .soc_tw, .soc_gg, .soc_in, .soc_fl, .soc_pi, .soc_rs, .soc_em{
+  color: #fff;
+  background-color: #333333;
+  padding: 8px 7px 5px;
+  text-decoration: none;
+  font-size: 12px;
 }
+
 /* HEADER AREA
 –––––––––––––––––––––––––––––––––––––––––––––––––– */
 .headlogo{
   background: url('imgs/mastersprite.png') 0px 45px;
   height:45px;
   width:42px;
-  margin: 16px 12px 0 0;
+  margin: 16px 4px 0 0;
 }
-header{
+.pageHeader{
   border: 13px solid #eeeeee;
   border-width: 0px 0px 13px;
   -moz-border-image: url(imgs/break.png) 0 0 13 repeat;
   -webkit-border-image: url(imgs/break.png) 0 0 13 repeat;
   -o-border-image: url(imgs/break.png) 0 0 13 repeat;
   border-image: url(imgs/break.png) 0 0 13 fill repeat;
-  padding-bottom: 60px;
+  padding-bottom: 30px;
 }
 .logoarea{
   width:260px;
   margin: -10px auto 22px;
   float: none;
 }
+.pageHeader h1{
+  margin: 16px -5px 3px 0;
+}
+h2{
+  margin: -12px 0 0 58px;
+  color: rgba(0,0,0,0.8);
+}
 /* BODY AREA
 –––––––––––––––––––––––––––––––––––––––––––––––––– */
 article{
   margin: 40px auto;
 }
+article h1{
+  margin: 25px 0 0 2px;
+}
+article h2{
```

Comment:
I would still avoid the selection based on a basic 'article' selector rather than a more specific class.